src/Features/CSharp/Portable/CodeFixes/ImplementAbstractClass/ImplementAbstractClassCodeFixProvider.cs

```diff
@@ -73,7 +73,7 @@ public sealed override FixAllProvider GetFixAllProvider()
         // internal for testing purposes.
         internal static string GetCodeActionId(string assemblyName, string abstractTypeFullyQualifiedName)
         {
-            return CSharpFeaturesResources.ImplementAbstractClass + ";" +
+            return CSharpFeaturesResources.Implement_Abstract_Class + ";" +
```

Comment:
Because it corresponds to the actual string ```Implement Abstract Class```.  Note: i'm not changing any actual loc strings, just the IDs for them.